class oop.py

```diff
@@ -0,0 +1,79 @@
+class Critter(object):
+    """ The virtual pet """
+    def __init__(self, name, hunger=0, boredom=0):
+        self.name = name
+        self.hunger = hunger
+        self.boredom = boredom
+
+    def __pass_time(self):
+        self.hunger += 1
+        self.boredom += 1
+
+    @property
+    def mood(self):
+        unhappiness = self.hunger + self.boredom
+        if unhappiness < 5:
+            m = 'happy'
+        elif 11 <= unhappiness <= 15:
+            m = 'frustrated'
+        else:
+            m = 'mad'
+        return m
+
+    def talk(self):
+        print('I\'m', self.name, 'and I feel', self.mood, 'now.\n')
```

Comment:
What does the output of this print statement look like?

Reply:
wdym

Reply:
What is the final result of this print statement.

Reply:
I'm (name) and I feel (mood) now

Reply:
Send me a screenshot of it, not you typing it

Reply:
texted it

Reply:
I didn't know that the `,` will automatically add spaces for you